components/external/js/three.js

```diff
@@ -33687,13 +33687,13 @@ THREE.ConvexGeometry = function( vertices ) {
 
 				for ( var e = 0; e < 3; e++ ) {
 
-					var edge = [ face[ e ], face[ ( e + 1 ) % 3 ] ];
+					var connector = [ face[ e ], face[ ( e + 1 ) % 3 ] ];
 					var boundary = true;
 
 					// remove duplicated edges.
 					for ( var h = 0; h < hole.length; h++ ) {
 
-						if ( equalEdge( hole[ h ], edge ) ) {
+						if ( equalConnector ( hole[ h ], connector ) ) {
```

Comment:
question remains unanswered
what is that file for?

Reply:
It was needed for something, but I don't know where it's used, I don't think anywhere. I'm not going to delete anything